src/viam/ur/module/ur_arm.cpp

```diff
@@ -189,6 +190,33 @@ enum class URArm::UrDriverStatus : int8_t  // Only available on 3.10/5.4
     DASHBOARD_FAILURE = 4
 };
 
+class URArmLogHandler : public urcl::LogHandler {
```

Comment:
any reason why the class is in the cpp file?

Reply:
fair but I vote for separating it, the file is already very long as is and it doesn't help readability

Reply:
moved to a new `utils.[c,h]pp`